src/library/blas/gens/clTemplates/trsv_gemv.cl

```diff
@@ -1420,7 +1420,8 @@ __kernel void %PREFIXtrsv_CLT_ComputeRectangle_kernel( 	__global const %TYPE* _A
 	int blkid = get_group_id(0);
 	int V= %V;
 
-	__local %TYPE solved[%TRIANGLE_HEIGHT];
+	__local %TYPE%V solvedV[%TRIANGLE_HEIGHT/%V];
+	__local %TYPE *solved = (__local %TYPE *)solvedV;
```

Comment:
This seems to be the only significant change.  Why the need for a local pointer to point inside the array?  Is the introduction of /%V a bugfix?